assets/monitoring/prometheus/v1/scylladb.servicemonitor.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   selector:
     {{- .endpointsSelector | toYAML | nindent 4 }}
-  jobLabel: scylla/cluster
+  jobLabel: app.kubernetes.io/name
```

Comment:
this needs to stay - this is the label from the service so in the end, if you name your scyllacluster scylla, it will make sure there is a job=scylla label used by some dashboards

Reply:
All of scylla-monitoring dashboards , including the serverless one, have "job" label hardcoded to either `scylla` or `scylla-manager`. This change breaks multiple metrics. What do you propose?

Reply:
Discussed internally. Temporarily fixed job labels in metrics used by dashboards. This is supposed to be fixed in upstream dashboards in long term.